README.md

```diff
@@ -202,12 +202,20 @@ For local development, running the ```mit``` harvester is recommended.
 
 Note: harvester-name is the same as the defined harvester "short name".
 
-Invoke a harvester a certain number of days back with the ```--days``` argument. For example, to run a harvester 5 days in the past, run:
+Invoke a harvester for a certain start date with the ```--start``` argument. Invole a harvester for a certain end date with the ```--end``` argument.
```

Comment:
Invole -> Invoke

and also this might be a better place to mention the `-s` `-e` variant